inc/class-rest-api.php

```diff
@@ -94,7 +94,7 @@ public function register_routes() {
 	 * REST requests.
 	 */
 	public function add_sst_request_filters() {
-		foreach ( get_post_types() as $post_type ) {
+		foreach ( get_post_types( [ 'show_in_rest' => true ] ) as $post_type ) {
```

Comment:
I think the answer is no but is there any chance that this would throw an error as undefined?

Reply:
Nope, this should be safe